Sources/TensorFlow/Layers/Convolutional.swift

```diff
@@ -37,8 +37,9 @@ public struct Conv1D<Scalar: TensorFlowFloatingPoint>: Layer {
     /// dilation and padding.
     ///
     /// - Parameters:
-    ///   - filter: The 3-D convolution kernel `[width, inputChannels, outputChannels]`.
-    ///   - bias: The bias vector `[outputChannels]`.
+    ///   - filter: The 3-D convolution filter of shape
+    ///     `[filter width, input channel count, output channel count]`.
```

Comment:
If this is a textual description, we should remove backquotes.

Reply:
Do this for all such filter/bias descriptions?

Reply:
How about you define `inputChannelCount` etc. as variables in the description of the layer, where you will also have equations in terms of them for computing the output width, and then you keep the backquotes and replace the descriptions with the variable names from earlier on?

Reply:
So, add properties in those struct definitions for `inputChannelCount` etc.? If so, using such variables to compute outputs wouldn't make much sense given that the `conv` bindings handle it under the hood and don't leak the details up into the Operators api.

Reply:
Sorry I was only referring to the documentation string. As in, name these quantities in the doc string and then add the equation for how the output shape is computed, again in the doc string.

Reply:
@eaplatanios When you get the chance could you please suggest a location in a doc string to add in the quantity names. I've update the source to remove backquotes and added the output computations. The only thing left is to introduce those quantities in the documentation. Thank you!